docker-compose.yaml

```diff
@@ -1,7 +1,7 @@
 version: "3"
```

Comment:
I'd go ahead and remove this line as well. Later docker compose versions warn about this line being deprecated

Reply:
Done.